src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -364,21 +364,22 @@ void Rules::sendWarningsForDisabledClusters()
         int nbScenariiForCluster = (int)scenariiForCurrentCluster.size();
         vector<uint>::iterator itv = scenariiForCurrentCluster.begin();
 
-        // Listing the 10 first scenarii numbers (years numbers) where current cluster is refered
-        // to. Notice that these scenarii could be less then 10, but are at least 1.
-        string listScenarii = to_string(*itv + 1);
+        // Listing the 10 first years for which the current cluster was given a specific TS number 
+        // in the scenario builder.
+        // Note that this list of years size could be less then 10, but are at least 1.
+        string listYears = to_string(*itv + 1);
         itv++;
-        for (; itv != scenariiForCurrentCluster.end(); itv++)
-            listScenarii += ", " + to_string(*itv + 1);
+        for (; itv != scenariiForCurrentCluster.end() && *itv + 1 <= 10; itv++)
```

Comment:
No message is displayed if a study contains e.g 50 years, but the 10 first years are disabled. I recommend using a container, then aggregate it into a string.

Reply:
You're right, changes done